addons/music/fnc_compileMusic.sqf

```diff
@@ -0,0 +1,40 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_compileMusic
+Description:
+A function used to gather a list of all usable music.
+Parameters:
+none
+Example:
+(begin example)
+_allmusic = [] call CBA_fnc_compileMusic
+(end example)
+Returns:
+config
+Author:
+Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+if (isNil QGVARMAIN(compiledMusic)) then {
+_cfg = configFile >> 'CfgMusic';
+_mission_cfg = missionConfigFile >> 'CfgMusic';
+_songs_unsorted = [];
```

Comment:
SQF usually uses camel case, not snake case and full words instead of abbreviations for variable names.
_cfg > _config
_mission_cfg > _missionConfig
_songs_unsorted > _unsortedSongs
I think that should be changed for consistency with the rest of the code.